project1/report.Rmd

```diff
@@ -76,5 +76,52 @@ data[, serum_cat := cut(serum_bilirubin, c(0, 1.1, 3.3, max(serum_bilirubin)))]
 # Code
 
 ```{r ref.label = knitr::all_labels(), echo = TRUE, eval = FALSE}
-# this R markdown chunk generates a code appendix
+# Reading in data
+data <- read.table("/Users/sktt1teddyfakerkhan/OneDrive - Emory University/bios521/project1_data-1.csv", 
+                   quote="\"", comment.char="")
+
+# Renaming variables
+colnames(data) <- c("caseid", "time", "status", "drug", "age", "sex", "ascites",
+                    "hepatomegaly", "spiders", "edema", "bilirubin", "cholesterol",
+                    "albumin", "urine_copper", "alk_phosphatase", "sgot", 
+                    "triglicerides", "platelets", "prothrombin", "hist_stage")
+
+# Combining censored cases
+data$dead <- ifelse(data$status == 0 | data$status == 1, 0, 1)
+table(data$dead)
+
+# Creating catergorical factor for bilirubin
+data$bili_cat <- ifelse(data$bilirubin < 1.1, "< 1.1", 
+                        ifelse(data$bilirubin >= 1.1 & data$bilirubin <= 3.3, "1.1-3.3", ">3.3"))
+data$bili_cat <- factor(data$bili_cat, levels = c("< 1.1", "1.1-3.3", ">3.3"))
+table( data$bili_cat)
+
+# Creating age in years
+data$ageinyear <- data$age/365.25
+
+## Analyses
+# Linear regression - excluding censored (univariate+multivariate)

```

Comment:
Could you put each code block in its own RMarkdown section, and label it clearly so that when the writers look at the `.pdf` they will know which plot corresponds exactly with which model? For example, the summary of `time ~ drug` would be labelled as "Regression with drug covariate" as the heading or title. That way we don't need to match up the order of these function calls with the produced plots, we can just reference the `.pdf`